spec/unit/provider/group/groupadd_spec.rb

```diff
@@ -7,6 +7,8 @@
     described_class.stubs(:command).with(:delete).returns '/usr/sbin/groupdel'
```

Comment:
It looks like both the spec files you've modified have changed permissions! That probably shouldn't have happened

Reply:
Sorry, my editor likes to set things 0755.